admin/app.js

```diff
@@ -49,8 +49,8 @@ const index = require('./routes/index')
 const testDeveloper = require('./routes/test-developer')
 const admin = require('./routes/admin')
 
-let app
-if (piping()) app = express()
+if (process.env.NODE_ENV !== 'production') piping()
```

Comment:
As per our offline review (@GuyHarwood, @GeorgeChatzigiannis)  - please can you change this check to look for 'development' in order to run `piping`